package.json

```diff
@@ -22,10 +22,15 @@
     "@babel/core": "^7.21.0",
     "@babel/node": "^7.20.7",
     "@babel/preset-env": "^7.20.2",
+    "@babel/register": "^7.21.0",
+    "chai": "^4.3.7",
+    "chai-http": "^4.3.0",
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
     "jest": "^29.4.3",
```

Comment:
what does this jest doing here when you have mocha and Chai ??

Reply:
i have fixed it